posthog/models/special_migration.py

```diff
@@ -8,6 +8,7 @@ class MigrationStatus:
     CompletedSuccessfully = 2
     Errored = 3
     RolledBack = 4
+    Starting = 5  # only relevant for the UI
```

Comment:
use a very high number here, so we don't get collisions if we add more states in the backend

Reply:
I mean, sure, but doesn't matter which way we add statuses. We can always add 6,7 and so on, even if they might have benefitted from a better logical order

Reply:
yes, the problem however is that we might be adding them to the backend PR first and then come here. We can change it in the future too in case of a collision I guess, but just seems easier to do it now.